### DISCUSSION TITLE
Questions on app with auto updates included, e.g. league-of-legends

### DISCUSSION THREAD

**Discussion by wyq977** [Homebrew/discussions#2125] [Casks] [answered]
Hi, everyone, Apps/Game like [league-of-legends](https://formulae.brew.sh/cask/league-of-legends) have a auto update upon each version. While the installer version stays the same (1.0) in [cask code](https://github.com/Homebrew/homebrew-cask/blob/1c1fd9caa0a3c408bc4c646fb8429bd208c62f5a/Casks/league-of-legends.rb#L2)

Riot also have update like bringing a Riot cient on macos so I assume some of the `zap trash` in the original cask code needs some rewrite.

For example like `com.riot.riotclient.checkinstalls` is not included in `zap uninstall`, [ref](https://docs.brew.sh/Cask-Cookbook#stanza-uninstall) using `"$(brew --repository homebrew/cask)/developer/bin/list_installed_launchjob_ids"`

Generally, how does one keep track of the Apps like these where they only release a installer on their website and ofter include major updates in their own auto update system like league-of-legends with homebrew.

Hope anyone could share some insights on these!
![Screen Shot 2021-09-12 at 4 11 52 PM](https://user-images.githubusercontent.com/27303600/132991187-af8de23d-ab8c-48f7-9c9d-44b008444e73.png)

Best

**Comment by zachauten** [comment, marked as answer]
There's an `auto_updates` [stanza in the cask dsl](https://docs.brew.sh/Cask-Cookbook#optional-stanzas) for apps that update themselves.
